include/deal.II/matrix_free/util.h

```diff
@@ -45,6 +49,55 @@ namespace internal
       return Quadrature<dim - 1>();
     }
 
+    template <int dim>
+    inline std::pair<ReferenceCell::Type, dealii::hp::QCollection<dim - 1>>
+    get_face_quadrature_collection(const Quadrature<dim> &quad,
+                                   const bool             do_assert = true)
+    {
+      if (dim == 2 || dim == 3)
+        for (unsigned int i = 1; i <= 3; ++i)
+          if (quad == Simplex::QGauss<dim>(i))
```

Comment:
Does this need to be adjusted now that we have more simplex quadrature formulas?

Reply:
Indeed it needs to be updated. Generally, I am not happy by the approach here but I don't have a better idea (that is too intrusive - making the face quadrature rule part of `MatrixFree::reinit()` just like in the case of `FEValues`). But, nevertheless let's proceed on this front and think resolve this issue till the release!